src/components/test/Outline.spec.js

```diff
@@ -17,7 +17,7 @@ const mockFunctionText = "mock function text";
 
 function generateDefaults(overrides) {
   return {
-    selectLocation: jest.genMockFunction(),
+    selectLocation: jest.fn(),
```

Comment:
`genMockFunction` was removed and was an undocumented API. Removing it from our examples and code.

https://github.com/facebook/jest/pull/6173